search/serializers.py

```diff
@@ -269,3 +312,27 @@ def serialize_comment_for_bulk(comment_obj):
     except NotFound:
         log.exception("Reddit comment not found: %s", comment_obj.id)
         raise
+
+
+def serialize_bulk_courses(ids):
+    """
+    Serialize courses for bulk indexing
+
+    Args:
+        ids(list of int): List of course id's
+    """
+    for course in Course.objects.filter(id__in=ids).iterator():
```

Comment:
An iterator is probably unnecessary overhead here: the batch size of `ids` will always be smaller than the batch size of the iterator (default is 2000) so it ends up incurring the overhead of a DB cursor and never ends up using it.